### PR TITLE
[IDSEQ-2731] Help Center link is always blue

### PR DESCRIPTION
**Pull request by ovalenzuela19**
# Description

Added the color `$black !important` to `cs.option` to overwrite the blue `cs.link` color.

# Tests
Verified that the changes were successful by extending the user drop down menu on the top right of the screen and confirming that the Help Center link was black and not blue.

### CODE REVIEW

File: app/assets/src/components/common/Header.jsx [hunk 1]
```diff
@@ -194,7 +194,8 @@ const AnnouncementBanner = ({ onClose }) => {
               }
             >
               Help Center
-            </ExternalLink>. Happy Holidays!
+            </ExternalLink>
+            . Happy Holidays!
```
Comment (ovalenzuela19) [inline app/assets/src/components/common/Header.jsx:198]: Looks like the linter doesn't want to take my changes :[

File: app/assets/src/components/common/Header.jsx [hunk 2]
```diff
@@ -261,15 +262,15 @@ const UserMenuDropDown = ({
       <BareDropdown.Item
         key="help"
         text={
-          <ExternalLink
+          <a
```
Comment (tfrcarvalho) [inline app/assets/src/components/common/Header.jsx:265]: We still want to use ExternalLink... you should be able to use the classname `option` (passed below) to apply the style you need.

**Comment by tfrcarvalho** [comment]
You can remove the sections you did not use and the italic sentence, which are meant as guidelines only.

File: app/assets/src/components/common/Header.jsx (hunk 2 above)
Comment (ovalenzuela19) [reply app/assets/src/components/common/Header.jsx:265]: @tfrcarvalho I'm not sure what you mean by being able to use the classname option to apply the style I need. Do you mean adding in a `color` property to cs.option?

Comment (tfrcarvalho) [reply app/assets/src/components/common/Header.jsx:265]: yes, for color you might need to use the `!important` ( as `color: $black !important;`) due to a legacy css.

Comment (ovalenzuela19) [reply app/assets/src/components/common/Header.jsx:265]: I'm reading more about the Classnames package and see that `cs.link` in `Link.jsx` component is overwriting the color to be` $primary-light` (blue)

Comment (tfrcarvalho) [reply app/assets/src/components/common/Header.jsx:265]: Yes, that is the default we want to enforce for all links. In this case, it is an exception and you want to overwrite that.

Comment (ovalenzuela19) [reply app/assets/src/components/common/Header.jsx]: > We still want to use ExternalLink... you should be able to use the classname `option` (passed below) to apply the style you need.

Just wondering, why do we still want to use ExternalLink?

**Pull request merged**